src/applications/static-pages/health-care-manage-benefits/modern-secure-messaging-page/components/App/index.js

```diff
@@ -0,0 +1,18 @@
+// Node modules.
+import React from 'react';
+
+// Relative imports.
+import { MhvSimpleSigninCallToAction } from 'applications/static-pages/mhv-simple-signin-cta';
+
+export const App = () => {
+  return (
+    <MhvSimpleSigninCallToAction
+      headingLevel="3"
+      serviceDescription="send secure messages"
+      linkText="send secure messages"
+      linkUrl="/my-health/secure-messages/inbox"
```

Comment:
Also, I think the URL is not expected to include `inbox`… does the app redirect to that URL?

```suggestion
      linkUrl="/my-health/secure-messages/"
```

Reply:
I used /inbox because I recalled Sarah asking for that url, lets mention in standup